app/assets/javascripts/components/alumni/_main.js.jsx

```diff
@@ -10,30 +10,20 @@ var Main = React.createClass({
   },
 
   render(){
-    var signedIn = this.props.signed_in;
-    var user = this.props.user;
-    var newPortfolios = this.state.newPortfolios;
-
-    if(user != null) {
-      return (
-        <div>
-          <SignedInNavbar user={user}/>
-          <MainJumbo/>
-          <MainSearch prepNewPortfolios={this.prepNewPortfolios} port_asc={this.props.port_asc} port_desc={this.props.port_desc} portfolios={this.props.portfolios} locations={this.props.locations}/>
-            <UserCard portfolios={newPortfolios}  avatars={this.props.avatars} bios={this.props.bios} locations={this.props.locations}/>
-          <Footer/>
-        </div>
-      )
-    } else {
-      return (
-        <div>
-          <GuestNavbar/>
-          <MainJumbo/>
-          <MainSearch prepNewPortfolios={this.prepNewPortfolios} port_asc={this.props.port_asc} port_desc={this.props.port_desc} portfolios={this.props.portfolios} locations={this.props.locations}/>
-          <UserCard portfolios={newPortfolios} avatars={this.props.avatars} bios={this.props.bios} locations={this.props.locations}/>
-          <Footer/>
-        </div>
-      )
+    if(this.props.user != null) {
+      var navbar = <SignedInNavbar user={this.props.user}/>
+    }
+    else {
+      var navbar = <GuestNavbar/>
```

Comment:
This is kind of just a style thing, but if I'm just testing for the existence of something, I'll use `if(!this.props.user)`

I also try to make all of my conditionals positive, so I'd probably further refactor it to be `if(this.props.user)` and swap my if/else.